docs/src/capi/reference/annotations.md

```diff
@@ -0,0 +1,12 @@
+# Annotations
+
```

Comment:
needs some text, we don't allow run on heading for several reason, but mainly because they look terrible. Suggestion - a short sentence explaining annotations and/or linking to upstream docs?